.github/pull_request_template.md

```diff
@@ -11,3 +11,13 @@ If an issue doesn't exist for this pull request (PR) to address, please open one
 ## Description
 
 -
+
+## Release notes
+
+<!--
+If this is related to a feature, bugfix or improvement, please add a summary of the change so it can be featured in the releaase notes.
```

Comment:
```suggestion
If this is related to a feature, bugfix or improvement, please add a summary of the change so it can be featured in the release notes.
```